### PR TITLE
Fix #4934:updated the readme.md file

### LINKED ISSUE #4934: README.md file is not updated.
**Describe the bug**
The opia android app is launch but the readme.md app says that it is not yet launch.

Screenshot from 2023-03-31 23-39-26

**Now as the android app is launched it must be updated as**

Screenshot from 2023-03-31 23-47-42

### PR DESCRIPTION
**Pull request by prafulbbandre**
## Explanation

Fixes #4934:Update readme.md file

## Essential Checklist

- [X] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [X] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [X] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [X] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by prafulbbandre** [comment]
@BenHenning can you approve this workflow.

**Comment by BenHenning** [comment]
The workflows are running now @prafulbbandre, but per the issue there's more work to do here yet. You should also update the branch to the latest develop changes, as well.

**Comment by prafulbbandre** [comment]
@BenHenning PTAL

File: .github/README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
 Oppia is an online learning tool that enables anyone to easily create and share interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
 
-The Android app is a new, not-yet-released frontend for Oppia that provides access to Oppia's curated numeracy lessons for users who may not have regular access to internet connectivity, and in a way that can be easily shared across multiple members of the same household.
+The Android app is frontend for Oppia that provides...household. The app is now available in beta and can be installed from the [Play Store](https://play.google.com/store/apps/details?id=org.oppia.android).
```
Comment (BenHenning) [inline .github/README.md]: The '...' means to keep the contents between the surrounding words ('provides' and 'household', in this case). You should re-add those words.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @prafulbbandre. Left one follow-up comment and synced with the latest develop. PTAL.

**Comment by prafulbbandre** [comment]
@BenHenning PTAL

File: .github/README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
 Oppia is an online learning tool that enables anyone to easily create and share interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
 
-The Android app is a new, not-yet-released frontend for Oppia that provides access to Oppia's curated numeracy lessons for users who may not have regular access to internet connectivity, and in a way that can be easily shared across multiple members of the same household.
+The Android app is frontend for Oppia that provides access to Oppia's curated numeracy lessons for users who may not have regular access to internet connectivity, and in a way that can be easily shared across multiple members of the same household. The app is now available in beta and can be installed from the [Play Store](https://play.google.com/store/apps/details?id=org.oppia.android).
```
Comment (BenHenning) [inline .github/README.md]:
```suggestion
The Android app is a frontend for Oppia that provides access to Oppia's curated numeracy lessons for users who may not have regular access to internet connectivity, and in a way that can be easily shared across multiple members of the same household. The app is now available in beta and can be installed from the [Play Store](https://play.google.com/store/apps/details?id=org.oppia.android).
```

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @prafulbbandre! Took another pass. Just had one nit--PTAL.

**Comment by BenHenning** [comment]
Also @prafulbbandre when addressing reviewer comments, please reply to the comment thread with an affirmative that you addressed the issue (e.g. "Done"), or with a follow-up indication of why you can't (e.g. because you're stuck & need clarification) or shouldn't make the change. Please don't mark other reviewers' comment threads as resolved, either, as reviewers use these to keep track of past comments when performing incremental follow-up reviews.

**Comment by prafulbbandre** [comment]
> Thanks @prafulbbandre! Took another pass. Just had one nit--PTAL.

@BenHenning Done.

File: .github/README.md (hunk 2 above)
Comment (prafulbbandre) [reply .github/README.md]: @BenHenning Done

**Review by BenHenning — APPROVED** [review]
Thanks @prafulbbandre! Apologies for the delayed review. The changes LGTM--just waiting on CI at this point (though I see no reason why it wouldn't pass, so enabling auto-merge as well).

**Comment by prafulbbandre** [comment]
Thanks @BenHenning

**Pull request merged**